[![memote tested](https://img.shields.io/badge/memote-tested-blue.svg?style=plastic)](https://biosustain.github.io/iVnat)

# iVnat: A genome-scale metabolic model of _Vibrio natriegens_

* Brief Model Description:

This repository contains a draft metabolic model of _Vibrio natriegens_.
For the most recent, comprehensive overview of this model's preformance check out [the memote snapshot report](snapshot.html)

* Model KeyWords:

**GEM Category:** Species; **Utilisation:** maximising growth; **Field:** metabolic-network reconstruction; **Type of Model:** draft; **Model Source:**; **Taxonomy:** _Vibrio natriegens_; **Metabolic System:** General Metabolism; **Bioreactor**

* Last update: 2018-07-03

* The model:

|Taxonomy | Template Model | Reactions | Metabolites| Genes |
|:-------:|:--------------:|:---------:|:----------:|:-----:|
|_Vibrio natriegens_||X|X|X|

This repository is administered by Christian Lieven ([@ChristianLieven](https://github.com/ChristianLieven)), Center for Biosustainability, Technical University of Denmark, Kgs. Lyngby 2800

## Installation

### Required Software:

* The scripts were generated using the following environment configuration. Importing the model has only been tested with [cobrapy](https://github.com/opencobra/cobrapy), which we recommend for any work with the model.

```
System Information
==================
OS           Windows 10
Python        3.6.0

Package Versions
================
XXX
```

### Contribute To Development
You have several options to contribute to and ultimately improve the metabolic model of _Vibrio natriegens_.

#### Provide Feedback (Simple)
1. Click on `Issues` in the top toolbar here on Github
2. Click on the green button `New issue` to open a new issue
3. Report errors or issues, provide feedback, or simply ask questions
4. Submit the issue by clicking `Submit new issue`

#### Participate (Advanced)
1. Fork the repository to your own Github account
2. Create a new branch from [`develop`](https://github.com/ChristianLieven/memote-m-capsulatus/tree/develop).
3. Make changes to the model
    + [cobrapy](https://github.com/opencobra/cobrapy) is highly recommended for making changes
    + Make a Pull Request to the `devel` branch, including changed `txt`, `yml` and `xml` files

## Complementary Scripts

*

## Complementary Data

*

## Contributors

* [Liza Radvanyi](www.github.com/radliz), Faculty of Information Technology and Bionics, Pazmany Peter Catholic University, Budapest
* [Christian Lieven](http://www.dtu.dk/english/service/phonebook/person?id=103199&tab=2&qt=dtupublicationquery) ([@ChristianLieven](https://github.com/ChristianLieven)), The Novo Nordisk Foundation Center for Biosustainability, Technical University of Denmark, Kgs. Lyngby, Denmark
* [Nikolaus Sonnenschein](http://www.biosustain.dtu.dk/english/service/phonebook/person?id=87972&tab=1) ([@phantomas1234](https://github.com/phantomas1234)), The Novo Nordisk Foundation Center for Biosustainability, Technical University of Denmark, Kgs. Lyngby, Denmark